docs/src/pages/en/themes/index.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Introduction to themes
+description: Understand how Media Chrome themes work and when you would want to build a theme
+layout: ../../../layouts/MainLayout.astro
+---
+
+Media Chrome provides us with [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) 
+that are easy to [style via CSS](./styling) which is great for media players 
+that are embedded in your own webpage or require less portability. However it's 
+often the case that a media player will be used by 3rd parties or 
+maybe the player needs to support different layouts and styles depending on
+the context.
+
+Themes provide a great solution for changing the look and feel of your player,
+and wrap your media controls up in a nice and portable package.
+
+## Basics
+
+Themes are created primarily with HTML + CSS and its contents are defined in a 
+[`<template>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template)
+element. 
+A shiny new web component called `<media-theme>` can then take the contents of
```

Comment:
Can you try rephrasing this? I'd recommend avoiding "shiny new" here, skipping unnecessary words while we learn the basics